packages/common/client-utils/api-report/client-utils.public.api.md

```diff
@@ -0,0 +1,19 @@
+## Public API Report File for "@fluid-internal/client-utils"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { EventEmitter } from 'events_pkg';
+import { IEvent } from '@fluidframework/core-interfaces';
+import { IEventProvider } from '@fluidframework/core-interfaces';
+import { IEventTransformer } from '@fluidframework/core-interfaces';
+import { TransformedEvent } from '@fluidframework/core-interfaces';
+
+export { EventEmitter }
+
+export { performance_2 as performance }
```

Comment:
Is the tool buggy? This doesn't look correct to me. There is no `performance_2`.

Reply:
I'll look into this. I've seen this pattern occur before and it's been innocuous (usually working around shadows of built-ins), but it definitely merits confirmation.

Reply:
Yeah, the import here matches what API-Extractor _would_ have generated in its rollups to avoid collisions with the `performance` built-in. What API-Extractor would have generated is:
```typescript
/**
 * This exported "performance" member masks the built-in globalThis.performance object
 * as an IsomorphicPerformance, which hides all of its features that aren't compatible
 * between Node and browser implementations.  Anything exposed on this performance object
 * is considered safe to use regarless of the environment it runs in.
 *
 * @internal
 */
declare const performance_2: IsomorphicPerformance;
export { performance_2 as performance }
```

Reply:
I think it's questionable that this is what should be in the API report, though. I'll file a bug for this.

Reply:
It looks like that line shouldn't even be listed. From the unified rollup:
```typescript
// @internal
const performance_2: IsomorphicPerformance;
export { performance_2 as performance }
```
So, it looks like there are cases where internal stuff is showing up in the public reports. Perhaps all of the renames are.

Reply:
Oh, nice catch! I wasn't looking at the tags. I'll play around with this a bit more to see if I can get a minimal repro for the bug.